frontend/src/lib/components/Support/supportLogic.ts

```diff
@@ -39,19 +40,21 @@ function getSentryLinks(user: UserType | null): string {
 }
 
 export const TARGET_AREA_TO_NAME = {
+    analytics: 'Product Analytics (Insights, Dashboards, Annotations)',
     app_performance: 'App Performance',
     apps: 'Apps',
-    login: 'Authentication (Login / Sign-up / Invites)',
     billing: 'Billing',
     cohorts: 'Cohorts',
     data_integrity: 'Data Integrity',
     data_management: 'Data Management',
     ingestion: 'Event Ingestion',
     experiments: 'Experiments',
     feature_flags: 'Feature Flags',
-    analytics: 'Product Analytics (Insights, Dashboards, Annotations)',
+    login: 'Login',
+    signup: 'Sign up / Invites',
```

Comment:
I think this makes sense BUT the list is already quite long, so in this case I think the benefits of disambiguation are not as large as the extra mental load required to scan the list. I propose we leave `login: 'Authentication (Login / Sign-up / Invites)'`